src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10103,6 +10150,7 @@ private static Precedence GetPrecedence(SyntaxKind op)
                 case SyntaxKind.ArrayCreationExpression:
                 case SyntaxKind.BaseExpression:
                 case SyntaxKind.CharacterLiteralExpression:
+                case SyntaxKind.CollectionCreationExpression:
```

Comment:
nit: please mention the precedence in the spec. Rex/Bill likely to ask question on that when they get to standardization.

Reply:
I can def add more.  But it's already in the spec with where this construct is in the grammar. (can't hurt to have more info though of course) :)